pkgs/misc/drivers/epson-201202w/default.nix

```
@@ -0,0 +1,74 @@
+{ lib, stdenv, fetchurl, rpmextract, autoreconfHook, file, libjpeg, cups }:
+
+  stdenv.mkDerivation rec {
+    version = "1.0.0";
+    filterVersion = "1.0.0";
+    pname = "epson-201202w";
```

these would traditionally be ordered like so:
```suggestion
    pname = "epson-201202w";
    version = "1.0.0";
    filterVersion = "1.0.0";
```